tests/models.py

```diff
@@ -0,0 +1,17 @@
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+OutOfSyncBase = declarative_base()
+
+
+class Widget(Base):
+    __tablename__ = "widgets"
```

Comment:
```suggestion
    __tablename__ = "widgets"

```

slightly more skimmable here to separate out the meta-level stuff from the columns